File: natureremocontroller/src/main/java/com/fishwaffle/natureremo/controller/models/SensorValue.java
/*
 * Copyright (c) 2018 FishWaffle.
 */

package com.fishwaffle.natureremo.controller.models;

import android.support.annotation.NonNull;

import java.io.Serializable;

/** センサー値 */
public class SensorValue  implements Serializable {
    /** センサー値 */
    public float val;
    /** 作成日時 */
    public String created_at;

    @NonNull
    @Override
    public String toString() {
        return "SensorValue{" +
                "val=" + val +
                ", created_at='" + created_at + '\'' +
                '}';
    }
}

File: natureremocontroller/src/main/java/com/fishwaffle/natureremo/controller/models/SensorValue.kt
/*
 * Copyright (c) 2019 FishWaffle.
 */

package com.fishwaffle.natureremo.controller.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import com.fishwaffle.natureremo.controller.NatureRemo.isIgnoreUnknown
import java.io.Serializable

/** センサーの値
 * @param value 値
 * @param created_at 作成日時
 *
 */
@JsonIgnoreProperties(ignoreUnknown = isIgnoreUnknown)
data class SensorValue(@JsonProperty("val") var value: Float, var created_at: String?) : Serializable